src/commands/staking/delegate/mod.rs

```diff
@@ -0,0 +1,104 @@
+use strum::{EnumDiscriminants, EnumIter, EnumMessage};
+
+mod deposit;
+mod deposit_and_stake;
+mod stake;
+mod stake_all;
+mod unstake;
+mod unstake_all;
+pub mod view_balance;
+mod withdraw;
+mod withdraw_all;
+
+#[derive(Debug, Clone, interactive_clap::InteractiveClap)]
+#[interactive_clap(input_context = crate::GlobalContext)]
+#[interactive_clap(output_context = DelegateStakeContext)]
+pub struct DelegateStake {
+    #[interactive_clap(skip_default_input_arg)]
+    /// What is validator account ID?
+    validator_account_id: crate::types::account_id::AccountId,
```

Comment:
We discussed it offline: it makes more sense to enter the user account first, not the staking pool account.

Reply:
Done